sonar-db/src/main/java/org/sonar/db/version/DatabaseVersion.java

```diff
@@ -29,7 +29,7 @@
 
 public class DatabaseVersion {
 
-  public static final int LAST_VERSION = 1125;
+  public static final int LAST_VERSION = 1126;
```

Comment:
I think we should have a gap in the version number when we change version to let space for a 5.5 migration